File: Service/Admin/RMA.Service.Admin.MasterDataManager.Contracts/Entities/ReferralStatusChangeAudit.cs
using RMA.Service.Admin.MasterDataManager.Contracts.Enums;

namespace RMA.Service.Admin.MasterDataManager.Contracts.Entities
{
    public class ReferralStatusChangeAudit
    {
        public int ReferralStatusChangeAuditId { get; set; }
        public int ReferralId { get; set; }
        public ReferralStatusEnum ReferralStatus { get; set; }
        public int ModifiedByUserId { get; set; }
        public System.DateTime ModifiedDate { get; set; }
    }
}

File: Web/RMA-Web-Shell/projects/shared-models-lib/src/lib/referrals/referral-status-change-audit.ts
import { ReferralStatusEnum } from "./referral-status-enum";

export class ReferralStatusChangeAudit {
    referralStatusChangeAuditId: number;
    referralId: number;
    referralStatus: ReferralStatusEnum;
    modifiedByUserId: number;
    modifiedDate: Date;
}
